.github/workflows/deploy-Extension.yml

```diff
@@ -0,0 +1,42 @@
+name: deploy-synopsys-extension
+
+on:
+  workflow_dispatch:
+
+jobs:
```

Comment:
can we add check index.js diff here as well

Reply:
added